react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { change: false };
```

Comment:
I recommend storing _which_ topic should be opened instead of this `change` thing. You can start your initial state as `null`, since nothing should be open.